tests/state_test_util.go

```diff
@@ -336,10 +336,14 @@ func MakePreState(db database.DBManager, accounts blockchain.GenesisAlloc, isTes
 	statedb, _ := state.New(common.Hash{}, sdb, nil, nil)
 	for addr, a := range accounts {
 		if len(a.Code) != 0 {
-			if isTestExecutionSpecState {
-				statedb.CreateSmartContractAccount(addr, params.CodeFormatEVM, params.Rules{IsIstanbul: true})
+			if _, ok := types.ParseDelegation(a.Code); ok {
+				statedb.SetCodeToEOA(addr, a.Code, params.Rules{IsIstanbul: true})
+			} else {
+				if isTestExecutionSpecState {
+					statedb.CreateSmartContractAccount(addr, params.CodeFormatEVM, params.Rules{IsIstanbul: true})
```

Comment:
How about using rules that are used throughout?
https://github.com/kaiachain/kaia/pull/149/files#diff-0f2a03d410f3b42fb5cc232db0be82b42a741b952aa3999887370ef91c1eca58R261

Reply:
I solved this at https://github.com/kaiachain/kaia/pull/149/commits/1e6561cdbbeafd1e50d33cd84aa8f00f7b83df93.